docs/tutorial.rst

```diff
@@ -89,8 +89,7 @@ With the virtual environment now activated, install cumulusci using pip:
 
     $ pip install cumulusci
 
-PyCrypto Error during install?  To resolve install Microsoft Visual Studio C++ 9.0 (http://aka.ms/vcpython27) then try the install again.
-Other Error? Check the error details in the console window for recommendations.
+Error during install? Check the error details in the console window for recommendations.
```

Comment:
"Got an error? read it." :/ 

> Errors during `pip install` are usually instructive, but there can be a lot of log, so make sure to scroll back for anything in red or important looking.

or something?

Reply:
I'll merge this and revise it in my next PR